integration/kubernetes/pod-liveness.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    test: liveness
+  name: liveness-exec
+spec:
+  containers:
+  - name: liveness
+    image: busybox
+    args:
+    - /bin/sh
+    - -c
+    - touch /tmp/healthy; sleep 6; rm -rf /tmp/healthy; sleep 12
```

Comment:
There are a bunch of timings, delays and sleeps in here. Would you be able to add a 1-line summary maybe of how the timing of the test works? Something like:
Launch->status check->sleep 3->kill->sleep 6->check dead etc.

And, do we actually use the healthy file or its output? not clear to me.